src/systems/tracked-controls-webvr.js

```diff
@@ -64,7 +64,11 @@ module.exports.System = registerSystem('tracked-controls-webvr', {
         this.el.emit('controllersupdated', undefined, false);
       }
     } catch (e) {
-      console.warn('A-Frame requires additional permissions to list the gamepads. If this is running in an IFRAME, you need to add `gamepads` to the `allow` attribute. If this is running as the top-level page, the HTTP `Permissions-Policy` header must not exclude this origin in the `gamepad` directive.', e);
+      if (e.name === 'SecurityError') {
```

Comment:
is this specific enough? We shouldn't check if we are in an iframe?

https://stackoverflow.com/questions/326069/how-to-identify-if-a-webpage-is-being-loaded-inside-an-iframe-or-directly-into-t

Reply:
No other code inside the try-catch can throw a SecurityError.